Core/OfficeDevPnP.Core/Pages/ClientSidePageCanvas.cs

```diff
@@ -1172,7 +1172,11 @@ private void LoadFromHtml(string html)
                     }
                     else if (controlType == typeof(ClientSideWebPart))
                     {
-                        var control = new ClientSideWebPart();
+                        var control = new ClientSideWebPart()
```

Comment:
Changes to this file are not related to the ReadSecurity issue. Why are they here?

Reply:
Not related, but important to get in. Just a mixup of PRs